cagallo/rancid-tomatillos
# Lint code ect

## What Features are Being Added?
- Installed eslint to linter our code.
- Created eslintrc file with list of rules

## What is Being Refactored?
- All .js were edited to account for any errors that popped up running eslint.

## Where should the reviewer start?
- No where in particular, all the .js files have been edited in some way.

## How should this be manually tested?
- Fetch the branch and run the site to make sure everything is still working properly. After that try running eslint and seeing whether any errors pop up. (At this point, on my local device, the linter was saying that there were no errors).

## What are the relevant tickets?
- Linting our code. 

## Any Other Notes/Questions/Problems?
- No that this time.
